content/data/unicorns.json

```diff
@@ -197,5 +197,22 @@
 		"profileImg": "./seanmiller.jpg",
 		"color": "#551a8b",
 		"roles": []
+	},
+ 	{
+		"id": "pierremtb",
+		"name": "Pierre Jacquier",
+		"firstName": "Pierre",
+		"lastName": "Jacquier",
+		"description": "Junior Hardware Engineer at Algolux. Computationally curious.",
```

Comment:
Side note: Love this tagline "Computationally curious" 🚀

Reply:
Awww thank you @crutchcorn